backend/timetable/models.py

```
@@ -0,0 +1,129 @@
+        Room,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="scheduled_classes",
+    )
+    day = models.CharField(max_length=20, choices=day_choices())
+    start_time = models.TimeField()
+    duration_slots = models.PositiveSmallIntegerField(default=1)
+
+    class Meta:
+        ordering = ["day", "start_time"]
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)